astroquery/mast/cutouts.py

```diff
@@ -31,49 +31,48 @@
 from ..exceptions import NoResultsWarning, InvalidQueryError, RemoteServiceError
 
 from . import conf
-from .utils import resolve_object
+from .utils import _parse_input_location
+from .core import MastQueryWithLogin
 
-__all__ = ["TesscutClass", "Tesscut"]
-
-
-def _parse_input_location(coordinates=None, objectname=None):
-    """
-    Convenience function to parse user input of coordinates and objectname.
-
-    Parameters
-    ----------
-    coordinates : str or `astropy.coordinates` object, optional
-        The target around which to search. It may be specified as a
-        string or as the appropriate `astropy.coordinates` object.
-        One and only one of coordinates and objectname must be supplied.
-    objectname : str, optional
-        The target around which to search, by name (objectname="M104")
-        or TIC ID (objectname="TIC 141914082").
-        One and only one of coordinates and objectname must be supplied.
-
-    Returns
-    -------
-    response : `~astropy.coordinates.SkyCoord`
-        The given coordinates, or object's location as an `~astropy.coordinates.SkyCoord` object.
-    """
 
-    # Checking for valid input
-    if objectname and coordinates:
-        raise InvalidQueryError("Only one of objectname and coordinates may be specified.")
-
-    if not (objectname or coordinates):
-        raise InvalidQueryError("One of objectname and coordinates must be specified.")
+__all__ = ["TesscutClass", "Tesscut"]
 
-    if objectname:
-        obj_coord = resolve_object(objectname)
 
-    if coordinates:
-        obj_coord = commons.parse_coordinates(coordinates)
+def _parse_cutout_size(size):
```

Comment:
Even as a private method, could this have documentation on what sorts of values `size` is allowed to be?

Reply:
Good catch! Did not even notice I neglected a docstring there.